tests/func/remote/test_setting_paths.py

```diff
@@ -0,0 +1,37 @@
+import os
+import posixpath
+import configobj
+
+import pytest
+
+from dvc.main import main
+from dvc.repo import Repo
+
+
+@pytest.mark.parametrize(
+    "field, remote_url",
+    [
+        ("credentialpath", "s3://mybucket/my/path"),
+        ("credentialpath", "gs://my-bucket/path"),
+        ("keyfile", "ssh://user@example.com:1234/path/to/dir"),
+        ("cert_path", "webdavs://example.com/files/USERNAME/"),
+        ("key_path", "webdavs://example.com/files/USERNAME/"),
+    ],
+)
+def test_relative_config_setting(tmp_dir, dvc, field, remote_url):
+    a_file = os.path.join("secrets", "file.txt")
+
+    remote_name = "some_name"
+    assert main(["remote", "add", "-d", remote_name, remote_url,]) == 0
```

Comment:
Linter CI job is complaining about this one
```suggestion
    assert main(["remote", "add", "-d", remote_name, remote_url]) == 0
```

Reply:
fixed